File: Easy/Temperatures/Temperatures.js
const n = parseInt(readline());
var inputs = readline().split(' ');
var mn = 10005;
if (n > 0)
    for (let i = 0; i < n; i++) {
        const t = parseInt(inputs[i]);
        if ((Math.abs(mn) > Math.abs(t)) || (Math.abs(mn) == Math.abs(t) && t > mn))
            mn = t
    }
else
    mn = 0
console.log(mn);

File: Easy/Temperatures/Temperatures.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Collections;
using System.Collections.Generic;

class Solution
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine()); // the number of temperatures to analyse
        string[] inputs = Console.ReadLine().Split(' ');
        int mn;
        if (n == 0) {
            mn = 0;
        }
        else {
            mn = int.Parse(inputs[0]);// a temperature expressed as an integer ranging from -273 to 5526
        }
        for (int i = 1; i < n; i++)
        {
            int t = int.Parse(inputs[i]);// a temperature expressed as an integer ranging from -273 to 5526
            if (t == 0) {
                mn = 0;
                break;
            }
            else if (Math.Abs(mn) == Math.Abs(t) && mn < 0) {
                mn = t;
            }
            else if (Math.Abs(mn) > Math.Abs(t)) {
                mn = t;
            }
        }
        
        Console.WriteLine(mn);
    }
}
